### PR TITLE
VOSTPT-29 load markers into the map

### PR DESCRIPTION
**Pull request by Lima21**

### CODE REVIEW

File: lib/domain/models/occurrence_model.dart [hunk 1]
```diff
@@ -16,7 +17,7 @@ abstract class OccurrenceModel
   int get code;
   @nullable
   String get updatedAt;
-
+  LatLng get coors;
```
Comment (Vanethos) [inline lib/domain/models/occurrence_model.dart]: What is `coors`?

File: lib/presentation/ui/home/home_page.dart [hunk 2]
```diff
@@ -239,32 +238,64 @@ class RecentListWidget extends StatelessWidget {
...
-                'accessToken': MAPBOX_ACCESS_TOKEN,
-                'id': 'mapbox.streets',
-              },
-            ),
-          ],
-        ),
+        StreamBuilder<List<OccurrenceModel>>(
+            stream: bloc.mockDataStream,
+            builder: (context, snapshot) {
+              if (!snapshot.hasData) {
```
Comment (Vanethos) [inline lib/presentation/ui/home/home_page.dart]: If there is a problem with the internet connection or if this takes a long time, then we are not showing a map here.

Why not show always the map and have that text above it OR a CircularProgressIndicator that shows that data is loading?

**Review by Vanethos — CHANGES_REQUESTED** [review]
Please view the requested changes

File: lib/domain/models/occurrence_model.dart (hunk 1 above)
Comment (Lima21) [reply lib/domain/models/occurrence_model.dart]: Is coordinates. Do you think that is better change to coordinates?

Comment (Vanethos) [reply lib/domain/models/occurrence_model.dart]: yes please 👍

**Review by Vanethos — APPROVED** [review]
Awesome 👍

**Pull request merged**